### PR TITLE
Introducing Code Of Conduct for Chaos Toolkit

### PR DESCRIPTION
**Pull request by pravarag**
Signed-off by: Pravar Agrawal pravarag@gmail.com

#### What this PR does / why we need it:
Defines Code of Conduct for Chaos Toolkit. Defines a way to express and codify the values defined in it to make Chaos Toolkit more welcoming and open for the community.

#### Checklist
- [x] Added CODE_OF_CONDUCT.md to the project

### CODE REVIEW

**Comment by Lawouach** [comment]
Thanks for that @pravarag! Will review that tomorrow :)

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Chaos Toolkit Code Of Conduct
+
+Chaos Toolkit adopts Debian project's Code Of Conduct for it's
```
Comment (Lawouach) [inline CODE_OF_CONDUCT.md]: s/it's/its ;)

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+participants and contributors.
+
+## Be respectful
+
+In ChaosToolkit project, inevitably there will be people with whom you
```
Comment (Lawouach) [inline CODE_OF_CONDUCT.md]: s/ChaosToolkit/the Chaos Toolkit

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (pravarag) [reply CODE_OF_CONDUCT.md]: Updated

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (pravarag) [reply CODE_OF_CONDUCT.md]: Updated

**Comment by Lawouach** [comment]
Thanks. I think it's good to go :)

**Comment by pravarag** [comment]
Thanks :) there goes my small contribution towards Chaos Toolkit

**Comment by Lawouach** [comment]
An important one!

**Pull request merged**